### PR TITLE
Overlay Lighting Color/Intensity Pass

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request

I was looking at screenshots of the game and realized we had a lot of light sources that were really... flat.

Medium intensity, not colored at all, cringe.

So I went over all the uses of overlay styled lighting (since I've done matrix lighting already) and gave them more unique features. Colors that match the sprite they're used with, intensity to produce vibes, that sort of thing.

It's kinda impossible to go one by one cause there's a LOT.

I may have gone a bit overboard with a few, I'm messing around with some things like giving bots colors based off their department, etc. We'll see how this all turns out.

Oh also I tweaked how the cone of overlay lighting is drawn. It seemed a bit too present to me so I dropped the alpha down from like 200 to 120 at max (so it's roughly half of the mask's alpha so it's less overwhelming

## Why It's Good For The Game

Lighting should be impactful, subtle and colorful

<details>
<summary>
Old Lights
</summary>

dreamseeker_QJ5bFZxd63
dreamseeker_DPVgI8wOoN
dreamseeker_pyEmTAb37x
dreamseeker_aDfvnea6YD
dreamseeker_RnkQSwmxmc
dreamseeker_G8NSC3lzAk
dreamseeker_yASfxtbHDR
dreamseeker_4ecBbG5urI
dreamseeker_udYlv8uAce
dreamseeker_bc2120Fnmr

</details>

<details>
<summary>
New Lights
</summary>

dreamseeker_0H11TyhGgx
dreamseeker_2B9AENHsfl
dreamseeker_3vOVRRMTSP
dreamseeker_5bTLup65rx
dreamseeker_iZzxZv4nfW
dreamseeker_Lhe9TSA0Av
dreamseeker_ngQJUv0tV4
dreamseeker_PL0z6sU7by
dreamseeker_xineFZDzPA
dreamseeker_zpXxbZZakS

</details>

## Changelog
:cl:
add: Tweaked the saturation, color and intensity of a bunch of lights
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
For context:
overlay lighting light_power scales from 0 to around 1.6 (I typically use 2 to mean full brightness)
1 is about 150/255 alpha

I don't really like it tbh but I cannot think of a better way to go about it

File: code/__HELPERS/paths/path.dm [hunk 1]
```diff
@@ -281,7 +281,7 @@
 	/// Are we being thrown?
 	var/thrown = FALSE
 	/// Are we anchored
-	var/anchored = FLASH_LIGHT_POWER
```
Comment (Kapu1178) [inline code/__HELPERS/paths/path.dm:284]: LMAO WHAT

Comment (LemonInTheDark) [reply code/__HELPERS/paths/path.dm:284]: listen ok I was relying on FALSE to autocorrect and I don't think it really did

**Comment by atlasle** [comment]
could you include screenshots? (like before and after) and also what do you think about having more grainy lighting? some other servers do it and it makes the lighting really not feel flat at all (compared to our transparent color gradient)

**Comment by EricZilla** [comment]
One more lighting PR and you're gonna have to change your name

**Comment by LemonInTheDark** [comment]
> could you include screenshots? (like before and after) and also what do you think about having more grainy lighting? some other servers do it and it makes the lighting really not feel flat at all (compared to our transparent color gradient)

I'll do a few, though I'm changing like 50 things so doin em all would be a bit much.

I don't think grainy lighting is a good idea tbh (Tho I've never played a grungelike codebase so idk maybe I misunderstand). I don't need the lights to be the sole focus of a scene, I just want them to have weight and depth. It's important to me that they don't fuck with the busyness of sprites too heavily too.

> One more lighting PR and you're gonna have to change your name

I'm like 30 in man we're far past that point

**Comment by LemonInTheDark** [comment]
Attached some examples

**Comment by SyncIt21** [comment]
Auto merge enabled 2 days ago!!!

Can you like update your branch with master and get this merged?

**Comment by LemonInTheDark** [comment]
Guh, unsure why screenshots are failing

**Comment by LemonInTheDark** [comment]
Ah I see

**Pull request merged**